apps/fm/test/unit/fm_test.js

```diff
@@ -2,10 +2,6 @@ requireApp('shared/js/airplane_mode_helper.js');
 requireApp('fm/js/fm.js');
 require('/shared/test/unit/load_body_html_helper.js');
 
-var PerformanceTestingHelper = {
-  dispatch: function() { }
-};
-
```

Comment:
Since you created this patch, kgrandon has updated this test file to make jshint happy and added a global declaration for PerformanceTestingHelper, so you'll need to rebase and remove that line, too.